stdlib/public/SwiftShims/CMakeLists.txt

```diff
@@ -133,10 +133,9 @@ swift_install_in_component(clang-builtin-headers
     DESTINATION "lib/swift/clang"
     PATTERN "*.h")
 
-swift_install_symlink_component(clang-resource-dir-symlink
-  LINK_NAME clang
-  TARGET ../clang/${CLANG_VERSION}
-  DESTINATION "lib/swift")
+swift_install_symlink(clang ../clang/${CLANG_VERSION}
+                      DESTINATION "lib/swift"
+                      COMPONENT clang-resource-dir-symlink)
```

Comment:
One fear of mine here is that you are using COMPONENT in a slightly different way than swift is using it. We should finish this transition to LLVM's way ASAP.